src/components/atoms/AtomStars.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="star-rating">
+    <font-awesome-icon v-for="i in 5" :key="i" :icon="getStarIcon(i)" />
+  </div>
+</template>
+
+<script>
+import { faStar as unfilledStar, faStarHalfAlt as halfFilledStar } from '@fortawesome/free-regular-svg-icons';
+import { faStar as filledStar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+
+export default {
+  name: "AtomStars",
```

Comment:
Change name to Stars (file as well)

Reply:
With this issue is current linter throws out following warning on build; `Component name "Stars" should always be multi-word  vue/multi-word-component-names`
And the style guide for Vue is also against one word component names - what ought I do?